about.html

```diff
@@ -0,0 +1,104 @@
+  <title>CompoundCode - door to door programming lessons.</title>
+  <link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Quicksand:wght@700&display=swap">
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Comfortaa:wght@300&family=Lato:wght@300&display=swap">
+  <link rel="stylesheet" href="css/index.css">
+</head>
+<body>
+  <aside class="desktop-only d-flex row top-mini-social-actions">
+    <a href="#"><img src="images/facebook-32.png" alt="Like us on Facebook"></a>
+    <a href="#"><img src="images/twitter-32.png" alt="Follow us on Twitter"></a>
+    <a href="#">English</a>
+    <a href="#">My Page</a>
+    <a href="#">Logout</a>
+  </aside>
+  <header class="d-flex cols wrap top-main-header is-mobile">
+    <a href="#" class="mobile-nav-trigger mobile-only" title=""><img src="images/menu-trigger-1.png" alt="Menu"></a>
+    <a href="index.html" class="desktop-only"><img src="images/compoundcode-logo-1.png" alt="Home"></a>
+    <nav class="nav-menu is-mobile d-flex cols">
+        <a href="#" class="close-nav-trigger mobile-only" title="">
+          <img src="images/close-menu.png" alt="Menu"> 
+        </a>
+        <a href="index.html" class="mobile-only">Home</a>
+        <a href="about.html" class="active wrap">About</a>
+        <a href="#">Program</a>
+        <a href="#">Join</a>
+        <a href="#">Sponsor</a>
+        <a href="#">News</a>
+        <a href="#" class="active-border">CC Campaign</a>
+    </nav>
+  </header>
+  <main class="d-flex cols">
+    <section class="d-flex cols full-width banner-content about-banner-section">
+        <div class="d-flex j-center full-width cols text-center">
+            <span class="greeting-text"> &lt; Hello! Sharing World / &gt; </span>
+            <h1>COMPOUNDCODE GLOBAL SUMMIT 2021</h1>
+        </div>
+        <p class="card-white-bg  text-center">
+            The Global Summit brings together the community of experts, 
+            academics, and activists who comprise the Creative Commons
+            Affiliate network in different country every two.
+            This year, we hope to Expand our invitation list 
+            including organizations and individuals
+        </p>
+    </section>
+    <span class="full-with text-center contact-per-mail general-text">
+        Please contact us per Email for any further questions about CC Global Summit 2021!<br>
+        <b>mwanzabj@gmail.com</b>
+    </span>
+    <section class="about-logo">
+        <h2 class="section-title text-center">CompoundCode Global Summit 2021 Logo</h2>
+        <hr class="section-dash-splitter">
+        <p class="general-text text-center">The logo of CC Global Summit 2015 was decided through the logo competition from 8. June to 7. July.</p>
+        <span class="logo">
+            <img src="images/compoundcode.png" alt="CompoundCode Logo">
+        </span>
+    </section>
```

Comment:
- [ ] Overall your fonts are not the good ones for mobile size. We can notice some screen overflows on your page, let's try to have a curated font for small devices. You can use some tools like [whatfonts](https://chrome.google.com/webstore/detail/whatfont/jabopobgcpjmedljpbcaablpmlmfcogm) to check the original font in a web page.

![Screenshot from 2021-10-14 08-44-33](https://user-images.githubusercontent.com/40503704/137274199-d280ecf0-d759-4453-8518-f5dab64f12ac.png)

Reply:
Actioned! 😃 

On to the last recommendation now 🤓